image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -234,6 +248,13 @@ spec:
       - hostPath:
           path: /
         name: host-root-ro
+    {{ if eq ._rox.env.openshift 4 }}
+      - hostPath:
+        path: /usr/share
+        name: usr-share-ro
+      - name: index-volume
```

Comment:
Just being curious here: Why do we need to mount an `emptyDir` volume at `/tmp`? Where does the name `index-volume` stem from?

Reply:
We're using an external library (ClairCore) to generate a (Scanner) v4.NodeIndex. 
The external lib copies the RPM package DB of the host and opens it, so it requires a writable folder to copy the DB to.
This is also why this is called the index-volume :smile: